chart/kubedb/templates/cluster-role.yaml

```diff
@@ -14,14 +14,13 @@ rules:
   resources:
   - customresourcedefinitions
   verbs:
-  - '*'
+  - "*"
```

Comment:
@tamalsaha Just a small cosmetic thing: It would be nice to be consistent at least in the same file, and use the same list format. This line uses yaml format, while most of the others using JSON.

Reply:
Fixed https://github.com/kubedb/cli/pull/319